lib/Dialect/FIRRTL/Transforms/Dedup.cpp

```diff
@@ -1169,7 +1169,8 @@ class DedupPass : public DedupBase<DedupPass> {
     // We must iterate the modules from the bottom up so that we can properly
     // deduplicate the modules. We have to store the visit order first so that
     // we can safely delete nodes as we go from the instance graph.
```

Comment:
Can you update the comment, maybe "We use early increment so that we can safely delete nodes from the instance graph as we go"

Reply:
Sounds great!